frontend/src/component/pages/shared/Header.scss

```diff
@@ -8,13 +8,22 @@ header {
   color: white;
   display: flex;
   justify-content: center;
+  position: relative;
 
   .center {
     display: flex;
     padding: 10px 0;
     width: $content-width;
   }
 
+  .sign-out-btn {
```

Comment:
Let's add a spacer between sign out button and the logo and set its `flex: 1`. It will automatically take up the space between logo and sign out button.